src/app.js

```diff
@@ -1,21 +1,21 @@
 import { h } from "./h"
 
-var lifecycleCallbackStack = []
+var callbackStack = []
```

Comment:
Just curious, why stack as opposed to queue?

Reply:
@pspeter3 A queue would imply we execute callbacks in the order we found them.

```counter() -> subcounter() -> subsubcounter()```

But instead we execute inside out, the deepest callback first and unwind.

```subsubcounter() -> subcounter() -> counter()```

So, AFAICT, the correct terminology is stack.

Reply:
Yeah, makes sense. I think we should document that as people may rely on the execution order.

Reply:
Execution order within siblings is undetermined though. I think this is not a problem, as it was with mixins because with modules the user, not the author, is responsible to give it a key / scope.